extensions/ql-vscode/src/remote-queries/sarif-processing.ts

```diff
@@ -0,0 +1,179 @@
+import * as sarif from 'sarif';
```

Comment:
Note there is already `extensions/ql-vscode/src/pure/sarif-utils.ts`. Does it make sense to move some of these functions over to that file? The other file can be renamed to `sarif-processing.ts` since that is a better name.

Reply:
I'd like to do that in a separate PR if that's okay - I noticed the file after I made my changes and I think there are potentially some conflicting function names etc.

Reply:
That's fine. There is some cleanup work to be done and it doesn't need to happen now.